README.md

```diff
@@ -80,11 +80,11 @@ Celo accepts a list of files as well as Glob patterns in both `encryption` and `
 
 ## Road map
 - [x] Use standar algorithm to encrypt files
-- [] Enhance file handling with buffers
-- [] Handle errors 
-- [] Show errors properly
-- [] Unit tests
-- [] Packaging
+- [ ] Enhance file handling with buffers
+- [ ] Handle errors 
+- [ ] Show errors properly
+- [ ] Unit tests
```

Comment:
I would say that Unit tests should be the next thing to do. What do you think about moving this to the top of the list?

Reply:
Of course, let me update.